resources/ory/values.yaml

```diff
@@ -1,2 +1,105 @@
+# Global or kyma related overrides
 global:
-  gateway: kyma-gateway
\ No newline at end of file
+  gateway: kyma-gateway
+  ingress:
+    domainName: kyma.local
+hpa:
+  oathkeeper:
+    minReplicas: 1
+    maxReplicas: 3
+    metrics:
+    - type: Resource
+      resource:
+        name: memory
+        targetAverageUtilization: 75
+
+# Overrides for the Hydra chart
+hydra:
+  ingress:
+    public:
+      enabled: false
+    admin:
+      enabled: false
+  hydra:
+    # The ORY Hydra configuration. For a full list of available settings, check:
+    #   https://github.com/ory/hydra/blob/master/docs/config.yaml
+    config:
+      serve:
+        public:
+          port: 4444
+        admin:
+          port: 4445
+        tls:
+          allow_termination_from:
+            - 10.0.0.0/8
+            - 172.16.0.0/12
+            - 192.168.0.0/16
+      secrets:
+        system: a3m0kCx6f87n332sPw1
+      urls:
+        self: {}
+      dsn: memory
+
+    autoMigrate: false
+    dangerousForceHttp: true
+    dangerousAllowInsecureRedirectUrls: false
+  resources:
+    limits:
+      cpu: 100m
+      memory: 256Mi
+    requests:
+      cpu: 50m
+      memory: 128Mi
+
+# Overrides for the Oathkeeper chart
+oathkeeper:
+  oathkeeper:
+    config:
```

Comment:
duplicated `oathkeeper` key, probably wrong copy paste?

Reply:
No, we target oathkeeper sub-chart here, and there's also "oathkeeper" section there, so from here the path looks like this.